scraper/src/youtube2zim/schemas.py

```diff
@@ -100,3 +100,10 @@ class Channel(CamelModel):
     joined_date: str
     collection_type: str
     main_playlist: str | None = None
+
+
+class Config(CamelModel):
+    """Class to serialize configuration data for the ZIM."""
+
+    main_color: str | None = "#000000"
+    secondary_color: str | None = "#FFFFFF"
```

Comment:
I think we should remove these two default values, they are never used in current codebase and could mislead a maintainer in believing that they are the default when no CLI argument is passed, while the default is in JS code.

Reply:
Removed in [35c4e76](https://github.com/openzim/youtube/pull/263/commits/35c4e767b329aacee1f51d67e8f3b15196163f97)